contracts/0.4.24/nos/NodeOperatorsRegistry.sol

```diff
@@ -592,7 +592,7 @@ contract NodeOperatorsRegistry is AragonApp, Versioned {
     /// @param _isTargetLimitActive active flag
     function updateTargetValidatorsLimits(uint256 _nodeOperatorId, bool _isTargetLimitActive, uint64 _targetLimit) external {
         _onlyExistedNodeOperator(_nodeOperatorId);
-        _auth(STAKING_ROUTER_ROLE);
+        _auth(MANAGE_NODE_OPERATOR_ROLE);
```

Comment:
Maybe we should have a test for this change?

Reply:
Added test for this role: https://github.com/lidofinance/lido-dao/pull/656/commits/46769765988bbed7b0d67945defddef52cf5d0cf

Reply:
:heart: